.gitignore

```diff
@@ -2,8 +2,7 @@
 npm-debug.log
 
 /coverage
-/dist
```

Comment:
Why we remove dist folder from here?

Reply:
Not sure, but you're right, `dist` should not be versioned. Updated pull request.